openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -862,3 +862,61 @@ export const calculatePercentage = (
 
   return round((numerator / denominator) * 100, precision);
 };
+
+/**
+ * Check if the color is a linear gradient
+ * @param color - Color string
+ * @returns {boolean} - True if the color is a linear gradient, false otherwise
+ */
+export const isLinearGradient = (color: string) => {
+  return color.toLowerCase().includes('linear-gradient');
+};
+
+/**
+ * Determines the safest container for rendering dropdown/popups like from Ant Design.
+ * It avoids containers that may clip the popup (e.g., with overflow: hidden),
+ * while still trying to keep the popup within a scrollable context if possible.
+ *
+ * @param trigger - The element that triggered the popup.
+ * @returns A suitable container element for rendering the popup.
+ */
+export const getVisiblePopupContainer = (
+  trigger?: HTMLElement
+): HTMLElement => {
+  // Fallback to document.body if no trigger is provided
+  if (!trigger) {
+    return document.body;
+  }
+
+  // Start from the trigger and walk up the DOM tree
+  let node: HTMLElement | null = trigger;
+
```

Comment:
Let's not use commonUtils, add specific utils to keep code seprate

Reply:
This can be added to all the dropdowns in future, hence added it in common utils.